rero_ils/translations/fr/LC_MESSAGES/messages.po

```diff
@@ -1016,10171 +930,5099 @@ msgstr ""
-msgstr "artificielles, autres langues"
+#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1_src.json:445
+#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json:445
+msgid "Canton"
+msgstr "Canton"
 
-#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1.json:771
-#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1.json:3214
-#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1.json:8874
-#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1_src.json:1495
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json:771
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json:3214
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json:8874
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json:1495
-msgid "arw"
-msgstr "arawak"
+#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1_src.json:453
+#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json:453
+msgid "Statements"
+msgstr ""
 
-#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1.json:775
-#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1.json:3218
-#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1.json:8878
-#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1_src.json:1499
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json:775
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json:3218
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json:8878
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json:1499
-msgid "asm"
-msgstr "assamais"
+#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1_src.json:458
+#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json:458
+msgid "Statement"
+msgstr ""
 
-#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1.json:779
-#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1.json:3222
-#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1.json:8882
-#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1_src.json:1503
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json:779
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json:3222
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json:8882
-#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json:1503
-msgid "ast"
-msgstr "asturien"
+#: rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1_src.json:459
+#: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json:459
+msgid "Statement of place and agent of the provision activity."
```

Comment:
Not sure I remember well, but shouldn't this be replaced by "Statement"?

Reply:
Yes. Done now.